.github/workflows/build.yml

```diff
@@ -7,6 +7,11 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v2
+            - uses: actions/setup-java@v1
+              with:
+                java-version: '1.8' # The JDK version to make available on the path.
+                java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+                architecture: x64 # (x64 or x86) - defaults to x64
```

Comment:
I looks like I used an older commit again, thats why its propably here.
But this shouldn't be a problem, right?

Reply:
Same for the changes in release.yml and build.gradle